perun-core/src/main/java/cz/metacentrum/perun/core/impl/ExtSourcesManagerImpl.java

```diff
@@ -207,16 +206,16 @@ private void updateExtSource(PerunSession sess, ExtSource extSource) throws Inte
 		}
 
 		// Check the attributes
-		if (!extSourceDb.getAttributes().equals(extSource.getAttributes())) {
+		if (!getAttributes(extSourceDb).equals(attributes)) {
 			log.debug("There is a change in the extSource attributes");
```

Comment:
I know, this is an unchanged log message, but can you add ExtSource object to tell, for which attributes has changed. Thanks.
```java
log.debug("There is a change in attributes for {}", extSource);
```